packages/edit-site/src/components/actions/view-actions.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { external } from '@wordpress/icons';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import { ACTION_TYPES } from './constants';
+
+export const viewPostAction = {
```

Comment:
Nit pick. My brain favors homogeneity, so I think it'd be happier if we did either 1) define all actions in one file or 2) define one action per file.

Reply:
I can do that. I'll update and move them to single file. If we need to, we can reconsider/split in the future.